dev/Cargo.toml

```diff
@@ -22,7 +22,7 @@ rcgen = "0.10.0"
 reqwest = { version = "0.11.11", features=["native-tls", "cookies"] }
 rustls-pemfile = "1.0.1"
 serde_json = "1.0.83"
-tokio = { version = "1.20.1", features = ["macros"] }
+tokio = { version = "1.20.1", features = ["full"] }
```

Comment:
Do you know what additional tokio features we're using? It would probably be better for compile times to list them explicitly.